File: 백준/Bronze/30802. 웰컴 키트/웰컴 키트.py
N = int(input())
sizes = list(map(int, input().split()))
T, P = map(int, input().split())

total = 0
for i in range(len(sizes)):
    total += (sizes[i] // T) + 1
    if sizes[i] % T == 0:
        total -= 1
print(total)

print(N//P, end=" ")
print(N%P)

File: 백준/Bronze/30802. 웰컴 키트/웰컴 키트.js
const input = require("fs")
  .readFileSync("/dev/stdin")
  .toString()
  .trim()
  .split("\n");

const N = Number(input[0]);
const sizeRequests = input[1].split(" ").map(Number);
const [T, P] = input[2].split(" ").map(Number);

const totalTShirtBundles = sizeRequests.reduce((sum, size) => {
  const bundles = Math.ceil(size / T);
  return sum + bundles;
}, 0);

const maxPenBundles = Math.floor(N / P);
const singlePens = N % P;

console.log(totalTShirtBundles);
console.log(maxPenBundles, singlePens);
